tests/integration/test.bulk_docs.js

```diff
@@ -67,6 +67,7 @@ adapters.forEach(function (adapter) {
               results[0].name.should.equal(
                 'conflict', 'First doc should be in conflict');
               should.not.exist(results[0].rev, 'no rev in conflict');
+              should.exist(results[0].id);
               for (i = 1; i < 5; i++) {
                 results[i].id.should.equal(i.toString());
```

Comment:
How in the heck was this even passing before? It looks like it's checking that each result has an id and it equals the document id.